tests/test_measurements.py

```diff
@@ -36,8 +37,9 @@ def test_create_using_validate_metrics(self):
 
     def test_str(self):
         ms = MeasurementSet(self.package, self.good,
-                            metric_set = self.metric_set)
-        expect = "test: {\ntest.cmodel_mag: 22.0 mag,\ntest.radius: 0.5 arcmin,\ntest.unitless: 22.0\n}"
+                            metric_set=self.metric_set)
+        expect = "test: {\ntest.cmodel_mag: 22.0 mag," + \
```

Comment:
You don't need the `+` here.

Separately, do we have an LSST-preferred multi-line continuator? There are several possibilities: wrapping in parens, appending `\`, triple-quotes.